profiles/Anjuli08.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/90701800/194690166-9327c711-0e96-4420-a1db-efbbe7807147.png",
+    "issueId": 1478,
+    "name": "Anjuli Aggarwal",
+    "username": "Anjuli08"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```